4.6.2/Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM microsoft/iis
+FROM microsoft/dotnet-framework:4.6.2
 
-RUN powershell -Command Add-WindowsFeature NET-Framework-45-ASPNET; \
-    powershell -Command Add-WindowsFeature Web-Asp-Net45; \
-    powershell -Command Remove-Item -Recurse C:\inetpub\wwwroot\*
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Comment:
I agree. It's a screwup on my part. I'm really sorry about that. In retrospect, pushing everyone to powershell as the default shell  should either be done in the servercore base image or not at all.